sip/sip-005-blocks-and-transactions.md

```diff
@@ -618,45 +618,19 @@ as follows:
 * A variable-length **payload**, of which there are five varieties.
 
 The _payload type ID_ can take any of the following values:
-* `0x00`:  the payload that follows is a **token-transfer payload**
+* `0x00`:  the payload that follows is a **stx-token-transfer payload**
```

Comment:
This wording here is intentional.  Alternative deployments of the Stacks chain (i.e. forks of Stacks on Bitcoin; app chains; ports to other blockchains) will have a different token ticker.  The SIP is meant to describe the system design, not the particular instance of it that uses STX.